### DISCUSSION TITLE
Ignoring specific analyzers

### DISCUSSION THREAD

**Discussion by wyardley** [aquasecurity/trivy#3755] [Q&A]
If I'm doing a scan for a specific type of config file (e.g., Terraform configs), is there a way to enable only that analyzer?

Similar to the `--scanners` option for `trivy fs`, but one level up?

I did try, e.g., `--file-patterns "kubernetes:^$"` to see if that would ignore all findings of type `kubernetes` (for example), but didn't seem to work.

**Comment by AnaisUrlichs** [comment]
Excuse my questions but are you keeping your terraform configuration and kubernetes manifests both in the same folder? I mean e.g.
main.tf
deployment.yaml
etc.
without subfolders? If yes, could you move them into different folders or what is the reason they are kept together?

**Comment by wyardley** [comment]
I’m doing recursive scans of sets of directories of IAC configs only.

However, some third party terraform modules I’m using contain Kube configs and dockerfiles incidentally

That’s all kind of besides the point; high level, it should IMO be possible to choose which types of config files to scan or not, and especially so if trivy is being proposed as a replacement for `tfsec`

**Reply by wyardley** [reply]
This is just one example, but I can think of tons of use cases where I’d only want to scan for one or two types of config.

**Reply by knqyf263** [reply]
We have the issue here. You can monitor it.
https://github.com/aquasecurity/trivy/issues/3987
